packages/nextjs/README.md

```diff
@@ -0,0 +1,83 @@
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br />
+</p>
+
+# Official Sentry SDK for Next.js
+
+[![npm version](https://img.shields.io/npm/v/@sentry/nextjs.svg)](https://www.npmjs.com/package/@sentry/nextjs)
+[![npm dm](https://img.shields.io/npm/dm/@sentry/nextjs.svg)](https://www.npmjs.com/package/@sentry/nextjs)
+[![npm dt](https://img.shields.io/npm/dt/@sentry/nextjs.svg)](https://www.npmjs.com/package/@sentry/nextjs)
+[![typedoc](https://img.shields.io/badge/docs-typedoc-blue.svg)](http://getsentry.github.io/sentry-javascript/)
+
+## Links
+
+- [Official SDK Docs](https://docs.sentry.io/quickstart/)
+- [TypeDoc](http://getsentry.github.io/sentry-javascript/)
+
+## General
+
+This package is a wrapper around `@sentry/node` for the server and `@sentry/react` for the client, with added functionality related to Next.js.
+
+To use this SDK, init it in the Sentry config files and import it in your code.
+
+```javascript
+// sentry.client.config.js
+
+import * as Sentry from '@sentry/nextjs';
+
+Sentry.init({
+  dsn: "__DSN__",
+  // ...
+});
+```
+
+```javascript
+// sentry.server.config.js
+
+import * as Sentry from '@sentry/nextjs';
+
+Sentry.init({
+  dsn: "__DSN__",
+  // ...
+});
+```
+
+```javascript
+// pages/index.js
+
+import * as Sentry from '@sentry/nextjs';
+```
+
+To set context information or send manual events, use the exported functions of `@sentry/nextjs`.
+Note that these functions will not perform any action before you have imported the SDK:
```

Comment:
```suggestion
```

This sentence sounds weird/stating the obvious, you can't reference methods like `Sentry.captureMessage` without importing `Sentry` first.